docs/samples/shards/FS/FileDialog/1.edn

```diff
@@ -0,0 +1,15 @@
+; pick any file
+(FS.FileDialog)
+(Log "file path")
+
+; pick any file from the parent directory
+(FS.FileDialog :CD "..")
```

Comment:
Can we be verbose here ? `CurrentDir` at least? this will be very visual soon. CD makes little sense to new users